src/reanimated2/layoutReanimation/web/elementManager.ts

```diff
@@ -0,0 +1,297 @@
+import type { TransformsStyle } from 'react-native';
+import { Animations, TransitionType } from '.';
```

Comment:
For me `Manager` suggest that this is a class with internal state and expose public API to deal with this state. Those files are more like a set of functions. - This is just my subject feel. I'm open to discussion.

Reply:
I agree. I've named it this way since this is how we originally called it. But that's true, I've also thought that using name with `manager` will sound more like a class. 

While we are on this topic, I considered creating such class, but when I saw `animationsManager` file that is responsible for native platforms I thought that we probably don't want to use that much of OOP (unlike in Gesture Handler)